webpack.common.config.js

```diff
@@ -2,39 +2,66 @@ const path = require('path');
 
 module.exports = {
   module: {
-    rules: [{
-      test: /\.jsx?$/,
-      exclude: /node_modules/,
-      use: 'babel-loader'
-    }, {
-      test: /\.css$/,
-      use: [
-        'style-loader',
-        'css-loader'
-      ]
-    }, {
-      test: /\.less$/,
-      use: [
-        'style-loader',
-        'css-loader',
-        {
-          loader: 'less-loader',
+    rules: [
+      {
+        test: /\.jsx?$/,
+         exclude: /node_modules/,
+        use: {
+          loader: 'babel-loader', 
           options: {
-            javascriptEnabled: true
+            plugins:
+              [
+                [
+                  'import', [
+                    {
+                      'libraryName': 'antd',
+                      'libraryDirectory': 'es',
+                      'style': true
```

Comment:
Please check [`.babelrc`](https://github.com/terrestris/react-geo/blob/master/.babelrc) file which already contains the similar import plugin config for `antd`. One of duplicates should be removed afterwards, IMO.

Reply:
Good catch! 

Actually I'm pro having the config placed inside the `.babelrc` otherwise the configuration would affect `webpack` [only](https://stackoverflow.com/a/43208353).

Reply:
I agree it makes more sense to have it in the babel setting. I will fix it.